src/Components/Layout/StockLayout.jsx

```diff
@@ -1,17 +1,17 @@
 import { Layout } from "antd";
 import React from "react";
+import "../Layout/StockLayout.css";
```

Comment:
Add a space after the imports. Also fix the relative path, edit out the unnecessary path.